boards/arm/b_l4s5i_iot01a/doc/index.rst

```diff
@@ -0,0 +1,233 @@
+    - 2x 16-bit basic
+    - 2x low-power 16-bit timers (available in Stop mode)
+    - 2x watchdogs
+    - SysTick timer
+- Up to 83 fast I/Os, most 5 V-tolerant
+- Memories
+    - Up to 2 MB Flash, 2 banks read-while-write, proprietary code readout protection
+    - Up to 640 KB of SRAM including 32 KB with hardware parity check
+    - External memory interface for static memories supporting SRAM, PSRAM, NOR and NAND memories
+    - Octo SPI memory interface
+- 4x digital filters for sigma delta modulator
+- Rich analog peripherals (independent supply)
+    - 1x 12-bit ADC 5 MSPS, up to 16-bit with hardware oversampling, 200 uA/MSPS
+    - 2x 12-bit DAC, low-power sample and hold
+    - 2x operational amplifiers with built-in PGA
+    - 2x ultra-low-power comparators
+- 18x communication interfaces
+    - USB OTG 2.0 full-speed, LPM and BCD
+    - 2x SAIs (serial audio interface)
+    - 4x I2C FM+(1 Mbit/s), SMBus/PMBus
+    - 6x USARTs (ISO 7816, LIN, IrDA, modem)
+    - 3x SPIs (4x SPIs with the Quad SPI)
+    - CAN (2.0B Active) and SDMMC interface
+    - SDMMC I/F
+    - DCMI camera interface
+- 14-channel DMA controller with multiplex request router
+- True random number generator
+- CRC calculation unit, 96-bit unique ID
+- AES and HASH hardware accelerators
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+
+
+More information about STM32L4S5VI can be found here:
+       - `STM32L4S5VI on www.st.com`_
+       - `STM32L4S5 reference manual`_
+
+
+Supported Features
+==================
+
+The Zephyr b_l4s5i_iot01a board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
```

Comment:
Please add sensors and BLE and wifi as they already supported

Reply:
added: 

| BLE       | module     | bluetooth                           |
+-----------+------------+-------------------------------------+
| WIFI      | module     | es-wifi                             |